x-pack/platform/plugins/shared/lens/public/react_embeddable/types.ts

```diff
@@ -401,8 +399,6 @@ export type LensApi = Simplify<
     HasLibraryTransforms<LensSerializedState, LensSerializedState> &
     // Let the container know the view mode
     PublishesViewMode &
-    // forward the parentApi, note that will be exposed only if it satisfy the PresentationContainer interface
-    Partial<HasParentApi<PresentationContainer>> &
```

Comment:
`LensApi` extends `DefaultPresentationPanelApi` which already includes `Partial<HasParentApi>`. You do not need to redefine `HasParentApi` here.